### ISSUE TITLE
Per week activity displays holidays on the wrong date.

### ISSUE THREAD

**Issue by Csardelacal** [Dolibarr/dolibarr#16452]
# Bug
Using a timezone other than GMT breaks holidays on `/projet/activity/perweek.php`. The holidays will be off by one day. In our example, using the timezone 'Europe/Berlin` yields the software calculating independence day on July 5th.

## Environment
- **Version**: 13.0.0
- **OS**: CentOS 7.0
- **Web server**: NGinx
- **PHP**: 7.3.21
- **Database**: MariaDB 10.4.6
- **URL(s)**: `/projet/activity/perweek.php`

## Expected and actual behavior
When setting the server timezone to Europe/Berlin creating independence day on July 4th, the overview should indicate that July 4th is a holiday. This works for Jan, 6th and any other date we tested.

The system displays July 5th

## Steps to reproduce the behavior
Holidays are all offset in our experience.

## Our solution
I am not submitting a pull, since I do not know whether this is the right way to fix this, but replacing the line 99 in `perweek.php` with 

```
$firstdaytoshow = dol_mktime(0, 0, 0, $first_month, $first_day, $first_year, true);
```

Would yield the system behaving properly. So I assume there's an issue with the fact that `dol_mktime()` uses server timezone by default but the block to calculate whether the day is a holiday does not.

**Comment by ksar-ksar** [comment]
Hello,

There were update on this in V13.0.1, could you try the update ?

**Comment by Csardelacal** [comment]
Note: I didn't notice that 13.0.1 was available, because the Github interface claims 13.0.0 is the latest.

I did pull the latest commit from the 13.0.0 branch and manually reviewed the changes, since the client currently has no version control running on the machine Dolibarr is installed on. But the changes I had to make to make the software usable were not in the repository.

When I applied the line in my OP (which is not in the 13.0 branch), the system stopped behaving properly when accounting the hours in the weekly overview. Since all other functions are not accounting for UTC time-zones.

I'll reach out to the customer and check if they are interested in trying 13.0.1 and will get back.
